rls/src/test/java/io/grpc/rls/SubchannelStateManagerImplTest.java

```diff
@@ -62,7 +62,7 @@ public void getAggregatedStatus_none() {
   @Test
   public void getAggregatedStatus_single() {
     for (ConnectivityState value : ConnectivityState.values()) {
-      if (value == ConnectivityState.SHUTDOWN) {
+      if (value == ConnectivityState.SHUTDOWN || value == ConnectivityState.TERMINATED) {
```

Comment:
I couldn't grok the semantics of `SubchannelStateManager`, whether the business logic needs to be changed or it's fair to treat SHUTDOWN and TERMINATED the same - the latter felt more correct based on my incomplete understanding of the class.